watchdogs.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/haya14busa/watchdogs/diff"
 )
 
+var TestNewExportedVar = 1 // TODO(#4): remove
```

Comment:
:dog:  [[watchdogs](https://github.com/haya14busa/watchdogs)] :dog:
exported var TestNewExportedVar should have comment or be unexported